R/construct_label.R

```diff
@@ -1,12 +1,14 @@
-#' Build labels with significances represented in bold or as raised 'a', and, if wanted, standard errors in brackets. Main usage for plotting tables and brace labels.
+#' Build labels with significances represented in bold or as raised 'a', and, if wanted, standard errors in brackets. Main usage for plotting tables and brace labels. #with significant values or significant labels? #whats a 'raised a' - 'superscripted a'?
 #'
 #' @inheritParams plot_lineplot
 #' @param dat Dataframe with the columns that should be merged into labels.
-#' @param label_sig_high_extra_column Logical indicating whether the superscript for significances should be added directly into the label (necessary for lineplots), or should be written into an extra column with the ending '_sig_superscript' (necessary for tables).
+#' @param label_sig_high_extra_column Logical indicating whether the superscript for significances should be added directly into the label (necessary for lineplots), or should be written into an extra column with the ending '_sig_superscript' (necessary for tables). #which is which?
```

Comment:
If `TRUE`, a new column will be build, if `FALSE` the superscript will be added directly into the label. Please rephrase so it becomes more clear.